ambry-commons/src/main/java/com/github/ambry/clustermap/HelixVcrUtil.java

```diff
@@ -283,15 +284,17 @@ private static boolean isDelayedRebalanceEnabled(String rebalancerClassName) {
   /**
    * A method to verify resources and partitions in src cluster and dest cluster are same.
    */
-  public static boolean isSrcDestSync(String srcZkString, String srcClusterName, String destZkString, String destClusterName) {
+  public static boolean isSrcDestSync(String srcZkString, String srcClusterName, String destZkString,
+      String destClusterName) {
 
     HelixAdmin srcAdmin = new ZKHelixAdmin(srcZkString);
     Set<String> srcResources = new HashSet<>(srcAdmin.getResourcesInCluster(srcClusterName));
     HelixAdmin destAdmin = new ZKHelixAdmin(destZkString);
     Set<String> destResources = new HashSet<>(destAdmin.getResourcesInCluster(destClusterName));
 
     for (String resource : srcResources) {
-      if (HelixVcrUtil.ignoreResourceKeyWords.stream().anyMatch(resource::contains)) {
+      String resourceLowercase = resource.toLowerCase();
+      if (HelixVcrUtil.ignoreResourceKeyWords.stream().anyMatch(resourceLowercase::contains)) {
```

Comment:
I would suggest to change `HelixVcrUtil` to ensure that `HelixVcrUtil.ignoreResourceKeyWords` are also lowercase. Maybe create a util method in `HelixVcrUtil` that converts `resource` to lowercase and answers the if predicate?

Reply:
+1